src/main/java/com/bhoruka/bloodbank/web/CampController.java

```diff
@@ -0,0 +1,57 @@
+package com.bhoruka.bloodbank.web;
+
+import com.bhoruka.bloodbank.model.request.CreateCampRequest;
+import com.bhoruka.bloodbank.model.response.CreateCampResponse;
+import com.bhoruka.bloodbank.service.CampService;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/camp")
+@Slf4j
+public class CampController {
+
+    private static final String CAMP_CREATE_SUCCESS_MESSAGE = "Camp created successfully.";
```

Comment:
There could be similar messages for other managers as well. Instead of being the respective class attributes, would it make sense to put these in an interface called BloodBankConstants and refer to the same? It might be more maintainable.

Reply:
Since these messages would only be applicable to their respective domains, taking them out of their class does not make sense. In case of reusable constants, I agree that we could take the approach you suggested.